src/app/TesseractMain.java

```diff
@@ -184,6 +184,59 @@ public void draw() {
 
   }//end render node
 
+  private void createBuiltInScenes() {
```

Comment:
all of this is now in the Util class now.  I couldn't do what I needed to do in Java so I moved it